### PR TITLE
Bug 1436499 - Spin up a dev treescriptworker.

### PR DESCRIPTION
**Pull request by Callek**

### CODE REVIEW

**Comment by Callek** [comment]
I manually created the sec groups in aws-console by copying beetmover's as-is.

File: configs/treescriptworker [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "us-east-1": {
+        "type": "treescriptworker",
+        "domain": "srv.releng.use1.mozilla.com",
+        "ami": "ami-18abe670",
+        "subnet_ids": ["subnet-33a98358"],
+        "security_group_ids": ["sg-03e4047e"],
+        "instance_type": "t2.medium",
```
Comment (escapewindow) [inline configs/treescriptworker]: Do you know if we need t2.medium, or are you just copying?
This matters especially if we have a 2nd idle instance.

Comment (Callek) [reply configs/treescriptworker]: This was just copying, but the initial clone of unified is a bit heavy and prone to timeout (locally) -- I could always make puppet pre-seed that on spin-up.

I'm ok with trying something smaller and increasing if we need to.

Comment (rail) [reply configs/treescriptworker]: IIRC, t2 can be used in VPC only. Otherwise it should work.

**Pull request merged**